modules/learn/pages/buckets-memory-and-storage/backend-storage.adoc

```diff
@@ -0,0 +1,65 @@
+= Backend Storage
+:page-pagination: 
```

Comment:
I don't think we should have Next and Previous page links in the Learn section, it should be more of an "every page is page one" environment. Also, it'll be inconsistent if this subsection has Next and Previous links, and none of the other subsections in the Learn section have them. However, you should definitely include links to related topics at the end of the page, if required.

Reply:
Yes, you're right. A few other places in the docs use a capital S, but the design docs all use lower case. I'll correct the other mentions while I'm here.